<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ClamGuard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="drop_shadow_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="dropshadowFrame">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 56, 59, 255));
border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="titleBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="titleFrame">
            <property name="font">
             <font>
              <family>SF Pro Display</family>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="title_label">
               <property name="font">
                <font>
                 <family>SF Pro Display</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 255, 38);</string>
               </property>
               <property name="text">
                <string>ClamGuard</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="titleButtons">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="minButton">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	border-radius: 8px;
	background-color: rgb(176, 109, 16);
}

QPushButton:hover
{
	
	background-color: rgb(255, 128, 0);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="closeButton">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(161, 0, 2);
}

QPushButton:hover
{
	
	background-color: rgb(255, 0, 4);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mainInterface">
         <property name="font">
          <font>
           <family>SF Pro Text</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QStackedWidget" name="stackedHome">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="pageHome">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QFrame" name="frame_content_home">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QFrame" name="scanFrame">
                   <property name="minimumSize">
                    <size>
                     <width>160</width>
                     <height>160</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>160</width>
                     <height>160</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QFrame
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 80px;
}
QFrame:hover
{
	border: 5px solid rgb(0, 255, 38);
}
	</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <widget class="QLabel" name="scanlabel">
                    <property name="geometry">
                     <rect>
                      <x>50</x>
                      <y>70</y>
                      <width>61</width>
                      <height>21</height>
                     </rect>
                    </property>
                    <property name="font">
                     <font>
                      <family>SF Pro Text</family>
                      <pointsize>15</pointsize>
                      <weight>75</weight>
                      <italic>false</italic>
                      <bold>true</bold>
                      <underline>false</underline>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QLabel
{
	color: rgb(0, 255, 38);
	border:none;
}

	</string>
                    </property>
                    <property name="text">
                     <string>SCAN</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageScan">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QFrame" name="frameScan">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="backButton">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>SF Pro Display</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 60px;
	color: rgb(0,255,38);
}
QPushButton:hover
{
	border: 5px solid rgb(0, 255, 38);
}
	</string>
                   </property>
                   <property name="text">
                    <string>Home</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="quickscanButton">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>SF Pro Display</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 60px;
	color: rgb(0,255,38);
}
QPushButton:hover
{
	border: 5px solid rgb(0, 255, 38);
}
	</string>
                   </property>
                   <property name="text">
                    <string>Quick Scan</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="fullscanButton">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>SF Pro Display</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 60px;
	color: rgb(0,255,38);
}
QPushButton:hover
{
	border: 5px solid rgb(0, 255, 38);
}
	</string>
                   </property>
                   <property name="text">
                    <string>Full Scan</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="customscanButton">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>SF Pro Display</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 60px;
	color: rgb(0,255,38);
}
QPushButton:hover
{
	border: 5px solid rgb(0, 255, 38);
}
	</string>
                   </property>
                   <property name="text">
                    <string>Custom Scan</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="cancelscanButton">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>120</width>
                     <height>120</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>SF Pro Display</family>
                     <pointsize>11</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton
{
	border: 5px solid rgb(0, 182, 24);
	border-radius: 60px;
	color: rgb(0,255,38);
}
QPushButton:hover
{
	border: 5px solid rgb(255,0,4);
	color: rgb(255,0,4);
}
	</string>
                   </property>
                   <property name="text">
                    <string>Cancel Scan</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="scanStatus">
                <property name="minimumSize">
                 <size>
                  <width>765</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>765</width>
                  <height>300</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPlainTextEdit
{
	 
	background-color: rgb(33, 33, 33);
	color: rgb(0,255,38);
}
	</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="statusBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="databaseVer">
            <property name="font">
             <font>
              <family>SF Pro Display</family>
              <pointsize>8</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 255, 38);</string>
            </property>
            <property name="text">
             <string>Database Version: </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
